.pre-commit-config.yaml

```diff
@@ -1,10 +1,11 @@
 default_install_hook_types: [pre-commit, pre-push]
 repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.2.1
+    rev: v0.5.7
```

Comment:
Noting for posterity that ruff is now up to v0.7.1 - but here i'm just syncing this to this repo's version